pages/common/git-tag.md

```diff
@@ -22,3 +22,7 @@
 - Get updated tags from upstream:
 
 `git fetch --tags`
+
+- List tags that contain a given commit:
```

Comment:
IIUC this lists all tags whose ancestors include the given commit (also, I'm not sure that you can't add more than one tag to a commit). But I'll re-check.